testing/square_bracket_accessor_test.py

```diff
@@ -73,153 +73,137 @@ def test_string_utf8(self):
         ])
 
-    @unittest.expectedFailure
     def test_chaining_double_quotes(self):
         res = mldb.query("""
             SELECT a["f"]["b"] FROM (SELECT {f: {b: 123}} AS a)""")
         self.assertEqual(res, [
-            ['_rowName', "a[f][b]"],
+            ['_rowName', "a.f.b"],
             ['result', 123]
         ])
 
-    @unittest.expectedFailure
     def test_mixing_no_quotes(self):
         res = mldb.query("SELECT a.f[b] FROM (SELECT {f: {b: 123}} AS a)")
         self.assertEqual(res, [
-            ['_rowName', 'a.f[b]'],
+            ['_rowName', 'a.f.b'],
             ['result', 123]
         ])
 
         res = mldb.query("SELECT a[f].b FROM (SELECT {f: {b: 123}} AS a)")
         self.assertEqual(res, [
-            ['_rowName', 'a.f[b]'],
+            ['_rowName', 'a.f.b'],
             ['result', 123]
         ])
 
-    @unittest.expectedFailure
     def test_mixing_double_quotes(self):
         res = mldb.query(
             """SELECT a.f["b"] FROM (SELECT {f: {b: 123}} AS a)""")
         self.assertEqual(res, [
-            ['_rowName', 'a.f[b]'],
+            ['_rowName', 'a.f.b'],
             ['result', 123]
         ])
 
         res = mldb.query(
             """SELECT a["f"].b FROM (SELECT {f: {b: 123}} AS a)""")
         self.assertEqual(res, [
-            ['_rowName', 'a.f[b]'],
+            ['_rowName', 'a.f.b'],
             ['result', 123]
         ])
 
-    @unittest.expectedFailure
     def test_outter_whitespaces_no_quotes(self):
         res = mldb.query("SELECT a[ foo   ] FROM (SELECT {foo: 123} AS a)")
         self.assertEqual(res, [
-            ['_rowName', "a[foo]"],
+            ['_rowName', "a.foo"],
             ['result', 123]
         ])
 
-    @unittest.expectedFailure
     def test_outter_whitespaces_double_quotes(self):
         res = mldb.query(
             """SELECT a[ "foo"   ] FROM (SELECT {foo: 123} AS a)""")
         self.assertEqual(res, [
-            ['_rowName', "a[foo]"],
+            ['_rowName', "a.foo"],
             ['result', 123]
         ])
 
-    @unittest.expectedFailure
     def test_inner_whitespaces_no_quotes(self):
-        res = mldb.query("""
-            SELECT a[f b] FROM (
-                SELECT parse_json('{"f b" : 123}') AS a
-            )
+        with self.assertRaises(mldb_wrapper.ResponseException):
+            mldb.query("""
+                SELECT a[f b] FROM (
+                    SELECT parse_json('{"f b" : 123}') AS a
+                )
                          """)
-        self.assertEqual(res, [
-            ['_rowName', "a[f b]"],
-            ['result', 123]
-        ])
-    @unittest.expectedFailure
+
     def test_inner_whitespaces_double_quotes(self):
         res = mldb.query("""
             SELECT a["f b"] FROM (
                 SELECT parse_json('{"f b" : 123}') AS a
             )
                          """)
         self.assertEqual(res, [
-            ['_rowName', "a[f b]"],
+            ['_rowName', "a.f b"],
```

Comment:
Ok never mind, MLDB returns that already when we do something like

`SELECT a."f b"`